components/sock_utils/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [0.2.0](https://github.com/espressif/esp-protocols/commits/sock_utils-v0.2.0)
+
+### Features
+
+- Declare socketpar and gai-error via standard headers ([7f103572](https://github.com/espressif/esp-protocols/commit/7f103572))
```

Comment:
```suggestion
- Declare socketpair and gai-error via standard headers ([7f103572](https://github.com/espressif/esp-protocols/commit/7f103572))
```

Reply:
Thanks for noticing, fixed separately (since this is a generated file).